recipes/ipyfilechooser/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ipyfilechooser" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 705a8a0d36b87414a72ab81a0327d3201a85e8f68ba1c70aa4f0250e1192a0db
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - ipywidgets
```

Comment:
It does not look like `ipywidgets` is needed to build the package so it can likely be removed from the `host` section

Reply:
I'm new to conda-forge packaging. Indeed ipyfilechooser doesn't require ipywidgets to be present prior to installing. It'll pull that in automatically upon installation via pip. Would you prefer I update the PR and remove it?

Reply:
yes, please remove it from the `host` section. It's only required under `run`.

Reply:
Change has been pushed.